### PR TITLE
Release v2.0.0

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
https://issues.folio.org/browse/UINV-125

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,15 +73,16 @@
     "okapiInterfaces": {
       "acquisitions-units": "1.1",
       "batch-group": "1.0",
+      "batch-voucher.export-configurations": "1.0",
       "configuration": "2.0",
       "finance.funds": "1.0",
-      "invoice": "3.0 3.1",
+      "invoice": "3.3",
```
Comment (aliaksei-chumakou) [inline package.json]: I don't see such interface https://github.com/folio-org/mod-invoice/blob/master/descriptors/ModuleDescriptor-template.json
there is no reason to make any change for minor versions. And this particular case will stop build of release artifact

File: package.json [hunk 2]
```diff
@@ -73,15 +73,16 @@
...
       "order-lines": "1.0",
       "organizations-storage.organizations": "3.0",
       "tags": "1.0",
-      "users": "15.0 15.1",
+      "users": "15.0",
```
Comment (aliaksei-chumakou) [inline package.json]: it's very questionable. What is the reason?
 I see 2 options here:

1. to leave as is 
2. to leave 15.1

Comment (AndreiShumski2) [reply package.json]: Left `15.1`

File: package.json (hunk 1 above)
Comment (AndreiShumski2) [reply package.json]: left `3.2`

**Pull request merged**